### ISSUE TITLE
Make pyproject bootstrap a separate command

### ISSUE THREAD

**Issue by johnslavik** [coherent-oss/coherent.cli#5]
It would be very useful to triage what `pyproject.toml` is produced and allow maximum composability.
The `install` command essentially produces the `pyproject.toml` temporarily to just run a command with it.
However, I'd like to inspect the produced `pyproject.toml` to facilitate development & debugging.

@jaraco do you see this as worth implementing? I'll do it by hand.

**Comment by johnslavik** [comment]
I'm seeing it as a twofold command similar to those in git (like `git rebase`+`git rebase --continue`).

Imagine this:
```bash
coh bootstrap .  # produced the bootstrap files
<arbitrary commands>
coh bootstrap --clean .  # or any fitting naming
```

**Comment by jaraco** [comment]
Yes, that seems reasonable. Let's start with just `coh bootstrap` for now, which will simply write out the pyproject.toml, probably by calling `__enter__()` on [`write_pyproject()`](https://github.com/coherent-oss/coherent.build/blob/6564c6a1370a0f519d25f39002e0858f08245e2a/bootstrap.py#L8) (and never calling `__exit__`). I'm tempted to say that it doesn't even need to take any parameters (let the caller clean it up with `git clean -fx` or delete it manually or commit it to their repo, as a lot of projects do now).
